buildout.cfg

```diff
@@ -1,79 +1,27 @@
 [buildout]
-index = https://pypi.org/simple/
-extends = https://dist.plone.org/release/5.2-latest/versions.cfg
-find-links =
-    https://dist.plone.org/release/5.2-latest/
-    https://dist.plone.org/thirdparty/
+extends = buildout.base.cfg
 
-parts =
-    instance
+package-name = senaite.core
+
+parts +=
     console_scripts
-    test
-    omelette
-    i18ndude
-    zopepy
     update_translations
     update_sources
-    write_code_headers
     write_contributors
 
-eggs =
-    senaite.core
+# Auto-checkout all sources but senaite.core
+auto-checkout =
     senaite.app.listing
     senaite.app.spotlight
     senaite.app.supermodel
     senaite.impress
     senaite.jsonapi
     senaite.lims
```

Comment:
Does this mean that extending packages need to define `auto-checkout = *`

Reply:
`buildout.base.cfg` defines `auto-checkout=*` already. We only override here because we don't want mr.developer to download `senaite.core` but rely on develop only. Rest lf products are not affected and don't need to override at all

Reply:
Ah, thanks! Overlooked that...